src/main/java/data/Shot.java

```diff
@@ -7,33 +7,17 @@
  * Created by martijn.
  * This class contains information about a Shot.
  */
-public class Shot {
-
-    // Counter that ensures no shots with duplicate numbers will be created.
-    private static int instanceCounter = 0;
+public abstract class Shot {
 
     // The name of the Shot.
     @Getter @Setter
     private String name;
 
-    // The description of the Shot.
-    @Getter @Setter
-    private String description;
-
-    // The number of the Shot.
-    @Getter
-    private int number;
```

Comment:
Both the DirectorShot and the CameraShot have this number field, why not keep it in the abstract class?